Cargo.toml

```diff
@@ -18,20 +18,29 @@ include = [
 ]
 
 [features]
-default = [ "gpu-allocator-feature" ]
+default = [ "gpu-allocator-feature", "egui-winit/default" ]
 gpu-allocator-feature = [ "gpu-allocator" ]
 
+## Underlying egui-winit features
+accesskit = ["egui-winit/accesskit"]
+android-game-activity = ["egui-winit/android-game-activity"]
+android-native-activity = ["egui-winit/android-native-activity"]
+bytemuck = ["egui-winit/bytemuck"]
+clipboard = ["egui-winit/clipboard"]
+links = ["egui-winit/links"]
+puffin = ["egui-winit/puffin"]
+serde = ["egui-winit/serde"]
+wayland = ["egui-winit/wayland"]
+x11 = ["egui-winit/x11"]
```

Comment:
So here I'm letting the user configure the underlying egui-winit features. This seems to be the typical cargo way of doing it: Pick expensive features of underlying libraries and expose them.